channels/py.m3u

```diff
@@ -1,3 +1,5 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/tdgaw2m.png" group-title="",RPC
 http://174.138.118.252/live/canal13stream_src.m3u8
+#EXTINF:-1 tvg-logo="https://i.imgur.com/572aYzz.jpg",Paraguay TV
+http://player-api.new.livestream.com/accounts/paraguaytv/events/8424381/live.m3u8
```

Comment:
Currently not working (tested with VLC and streamtest)

Reply:
Currently Working
https://streamtest.in/logs/http-player-api-new-livestream-com-accounts-paraguaytv-events-8424381-live-m3u8-58ef28ca-9eaf-4030-8c15-1e891b37ef93

Reply:
I think the stream is geo-blocked because for me it still does not work
<img width="1001" alt="Screenshot 2019-10-19 at 06 18 58" src="https://user-images.githubusercontent.com/7253922/67137080-5d5d5980-f238-11e9-932b-1b777bd398fe.png">

Reply:
Removed Link